<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(249, 249, 249);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>141</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>141</width>
      <height>141</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="WDifussion" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>421</width>
      <height>381</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>421</width>
      <height>381</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>421</width>
      <height>382</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(177, 177, 177);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>211</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>211</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

color: rgb(0, 0, 122);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Difusión&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRunSimulation">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>151</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 64);</string>
    </property>
    <property name="text">
     <string>Ejecutar Simulación </string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStopSimulation">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>230</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>151</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 64);</string>
    </property>
    <property name="text">
     <string>Detener Simulación </string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnToMenu">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 64);</string>
    </property>
    <property name="text">
     <string>Volver al Menú </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
